app/src/main/java/com/example/tmdb/screens/Router.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.tmdb.screens
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+
+sealed class Screen() {
+    data class MainScreen(val tab: MainScreenTab) : Screen()
+    object Details : Screen()
+}
+
+open class MainScreenTab() {
+    object HomeTab : MainScreenTab()
+    object FavoriteTab : MainScreenTab()
+}
+
+object Router {
```

Comment:
Since this app is simple enough, it is ok to use this router, but I recommend you to use compose navigation for your project or even replace it in homeworks, if you will have time to do so.